api/README.md

```diff
@@ -1,36 +1,59 @@
-# Setup
+# Bloom Backend
+
+![TypeScript](https://img.shields.io/badge/typescript-%23007ACC.svg?style=for-the-badge&logo=typescript&logoColor=white) ![NestJS](https://img.shields.io/badge/nestjs-%23E0234E.svg?style=for-the-badge&logo=nestjs&logoColor=white) ![Prisma](https://img.shields.io/badge/Prisma-3982CE?style=for-the-badge&logo=Prisma&logoColor=white) ![Postgres](https://img.shields.io/badge/postgres-%23316192.svg?style=for-the-badge&logo=postgresql&logoColor=white)
+
+## Overview
+
+This is the backend services container. Data is stored in a [Postgres](https://www.postgresql.org/) database and served over HTTPS to the frontend (either at build time for things that can be server-rendered, otherwise at run time). Most services are part of a [NestJS](https://nestjs.com/) application. [Prisma](https://www.prisma.io/) is the application's ORM. [OpenAPI Swagger](https://swagger.io/tools/swagger-ui/) documentation is automatically generated by the server at http://localhost:3100/api/ in local development environments, or in any other environment by adding `/api` to the api URL. This can be helpful to get a more visual overview of all available endpoints.
+
 ## Installation
-Make sure the .env file's db placement is what works for your set up, Then run the following:
 
-```bash
-$ yarn install
-$ yarn prisma generate
-$ yarn build
-$ yarn db:setup:staging
-```
+The following commands are for macOS / Linux, but you can find equivalent instructions for Windows machines online.
+
+### Environment Variables
+
+Configuration of the backend is pulled from environment variables defined in an `.env` file in the api directory. Copy the `.env.template` file in api into a `.env` file. Some keys are secret and are internally available. The template file includes default values and descriptions of each variable.
+
+### Dependencies
+
+If you don't have yarn installed, you can install homebrew with [these instructions](https://brew.sh/) and then do so with `brew install yarn`.
+
+#### Installing Node
+
+We are currently using Node version 18. You can install Node using homebrew with the following command: `brew install node@18`.
+
+If you have multiple versions of Node installed, you can use [nvm](https://github.com/nvm-sh/nvm) (node version manager), or other similar tools, to switch between them. Ensure you're on the right version by checking with `node -v`.
+
+If along the way you get `env: node: No such file or directory`, inspect the output from installing node for instructions on if you might need to add node to certain terminal paths.
+
+#### Installing Postgresql
+
+You can install Postgres using homebrew with the following command: `brew install postgresql@14`. You then start it with `brew services start postgresql@14`.
```

Comment:
nit: we currently have postgres 15 in our deployed instances, we might want to keep this in line with that

Reply:
🫡